scripts/zkai_range_disciplined_activation_receipt.py

```
@@ -0,0 +1,464 @@
+
+
+def validate_source_evidence(payload: Any) -> None:
+    if not isinstance(payload, dict):
+        raise RangeActivationReceiptError("source evidence must be an object")
+    if payload.get("schema") != SOURCE_SCHEMA:
+        raise RangeActivationReceiptError("source evidence schema drift")
+    if payload.get("decision") != SOURCE_DECISION:
+        raise RangeActivationReceiptError("source evidence decision drift")
+    conclusion = payload.get("conclusion")
+    if not isinstance(conclusion, dict):
+        raise RangeActivationReceiptError("source evidence lacks conclusion")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)